### DISCUSSION TITLE
Как получить ревью кода проекта?

### DISCUSSION THREAD

**Discussion by kirilinsane** [HowProgrammingWorks/LiveQA#21] [Q&A] [answered]
Куда присылать ссылку на проект для получении оценки за семестр?

**Comment by gapiyka** [comment]
issues check)

**Comment by tshemsedinov** [comment, marked as answer]
Добавить прямо сюда в виде discussion: https://github.com/HowProgrammingWorks/LiveQA/discussions/categories/code-review
